src/byte_array.rs

```diff
@@ -244,9 +245,9 @@ mod tests {
         ba.remove_range(1, 2);
         ba.sort(|a, b| a.cmp(b));
         unsafe { ba.set_size(3) };
-        assert_eq!(ba, "aab".as_bytes());
+        assert_eq!(ba, b"aab" as &[u8]);
```

Comment:
Maybe better use `&b"aab"[..]`

Reply:
I'm happy either way.  If you would prefer the `[..]` form over the `as &[u8]` form, then I'll change this, and other instances of this kind of change.  It's basically aesthetic so I shall defer to you.  Just say if you want me to change those up.

Reply:
Lets leave it as is

Reply:
Ok :+1:  Thanks for reviewing.